apps/web/src/server/lib/date-formatter.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * converts an ISO date string to Europe/London time zone date string
+ *
+ * @param {string} dateISOString
+ * @returns {string}
+ */
+export const dateFormatter = (dateISOString) =>
```

Comment:
There is a simila function on the API app... worth discussing if we could consolidate in some shared helpers